server/chat-plugins/chat-monitor.ts

```diff
@@ -567,9 +567,9 @@ export const commands: ChatCommands = {
 			}
 			saveFilters(true);
 			const output = `'${word}' was added to the ${list} list.`;
-			const upperStaff = Rooms.get('upperstaff');
-			if (upperStaff) upperStaff.add(output).update();
-			if (room.roomid !== 'upperstaff') return this.sendReply(output);
+			if (room?.roomid !== 'upperstaff') this.sendReply(output);
+			Rooms.get('upperstaff')?.add(output).update();
+			return this.sendReply(output);
```

Comment:
...I can't tell what you're trying to do here, but it clearly isn't what you did...

```suggestion
			Rooms.get('upperstaff')?.add(output).update();
			if (room?.roomid !== 'upperstaff') this.sendReply(output);
```